documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -1,5 +1,11 @@
 # Current Release notes
 
+## Version 8.11.0
+
+### New features
+
+* For Stateless and Rapid scans, the scanId and Detector tool being run, are now stored in the codeLocations section of the status.json file containing the run summary.
+
```

Comment:
We should also update status file documentation [here](https://sig-product-docs.synopsys.com/bundle/integrations-detect/page/runningdetect/status-file.html) (also maybe we could fix the _[solution name]_ not being resolved on that page while we're at it)

Reply:
Taking a look at that now... looks like [solution_name] has possibly never resolved when part of a property value descriptor in this file. I'll find a solution if I can under a new branch.